experiments/run_experiment.py

```diff
@@ -159,51 +164,64 @@ def validate_args(args: argparse.Namespace, parser: argparse.ArgumentParser):
         )
 
 
-def _get_dataset(
+def run_mrmc(
+    run_seed: int,
+    step_size: float,
+    confidence_cutoff: Optional[float],
+    noise_ratio: Optional[float],
+    rescale_ratio: Optional[float],
+    volcano_degree: float,
+    volcano_cutoff: float,
+    num_clusters: int,
+    max_iterations: int,
     dataset_name: str,
-) -> Tuple[pd.DataFrame, base_loader.DatasetInfo]:
-    """Gets the dataset. Useful for testing."""
-    return data_loader.load_data(data_loader.DatasetName(dataset_name))
+    model_type: str,
+    cluster_seed: int,
+    **_unused_kwargs: Any,
+) -> Tuple[Sequence[pd.DataFrame], pd.DataFrame]:
+    """Runs MRMC using the given configurations.
 
+    Args:
+        run_seed: The seed used in the run. All random numbers are derived from
+            this seed except the clustering, which uses cluster_seed.
+        step_size: The step size to use when rescaling the recourse.
+        confidence_cutoff: The target model confidence.
+        noise_ratio: The optional ratio of noise to add.
+        rescale_ratio: The optional ratio by which to rescale the direction.
+        volcano_degree: The degree to use in the MRM volcano alpha function.
```

Comment:
Would be worth adding `volcano_function(_name)` to explicitly specify which alpha function to use if we want to add this to hyperparam considerations.

Reply:
I think we discussed this in our last meeting and we will only use one alpha function (at least for now). I'm closing this as resolved, but please let me know if I'm mistaken!